app/views/results/index.html.erb

```diff
@@ -3,7 +3,14 @@
 <div class="govuk-width-container">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <h1 class="govuk-heading-xl">Teacher training courses</h1>
+      <% if params[:l] == "3" %>
```

Comment:
For readability, it might be useful to wrap this in a helper method. It's hard to tell what '3' represents